generator/src/conf/builtin-extensions

```diff
@@ -0,0 +1,52 @@
+@theia/vscode-builtin-bat@0.2.1
```

Comment:
hello, it seems that these extensions are 4months old ?

Reply:
Correct. 
I can use either `latest` which is `0.2.1` or `next` which is `0.3.0-next.XXXXXXX`.
Since it is mostly grammar I don't expect much difference and I can be sure that it works with `latest` tag.
In case of `next` I always can face unexpected issue like this https://github.com/eclipse-theia/theia/issues/6672